backend/monitoring/core_integration.py

```
@@ -0,0 +1,215 @@
+            # Get request body safely without consuming it
+            request_body = None
+            if method in ["POST", "PUT", "PATCH"]:
+                try:
+                    # Try to get body from request state if already read
+                    if hasattr(request.state, "_json"):
+                        request_body = str(request.state._json)
+                    elif hasattr(request.state, "_body"):
+                        request_body = request.state._body.decode("utf-8") if request.state._body else None
+                    # Note: We cannot read request.body() here as it would consume the stream
+                    # The body should be captured by endpoint handlers and stored in request.state
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)